tanner/utils/aiodocker_helper.py

```diff
@@ -0,0 +1,82 @@
+import aiodocker
+import logging
+
+from tanner.config import TannerConfig
+
+
+class AIODockerHelper:
+    def __init__(self):
+
+        self.logger = logging.getLogger('tanner.aiodocker_helper.AIODockerHelper')
+
+        self.docker_client = aiodocker.Docker()
+        self.host_image = TannerConfig.get('DOCKER', 'host_image')
+
+    async def setup_host_image(self, remote_path=None, tag=None):
+
+        try:
+            if remote_path and tag is not None:
+                params = {"tag": tag, "remote": remote_path}
+                await self.docker_client.images.build(**params)
+
+            image = await self.docker_client.images.list(filter=self.host_image)
+            if not image:
+                await self.docker_client.images.pull(self.host_image)
+
+        except aiodocker.exceptions.DockerError as docker_error:
+            self.logger.exception('Error while pulling %s image %s', self.host_image, docker_error)
+
+    async def get_container(self, container_name):
+        container = None
+        try:
+            container = await self.docker_client.containers.get(container=container_name)
+
+        except aiodocker.exceptions.DockerError as server_error:
+            self.logger.exception('Error while fetching %s container %s', container_name, server_error)
+        return container
+
+    async def create_container(self, container_name, cmd=None, image=None):
```

Comment:
This function, for now, is used only in tests, why you need it?

Reply:
Yes, because with aiodocker we can execute commands in both ways, by using `client.containers.create` and with `docker_client.containers.run` as well. But, its good to have if we need to create containers at some point then it can help :)